src/app/components/course/course-event-create/course-event-create.component.ts

```diff
@@ -81,11 +80,9 @@ export class CourseEventCreateComponent implements OnInit {
     }
 
     checkCanSubmit(): void {
-        if (this.eventName && this.eventType && this.startTime && this.endTime) {
```

Comment:
Now that you have touched this file you ought to refactor.
Please use angulars form control and have proper validators for the field rather than manually doing this.